server/forge/setup/setup.go

```diff
@@ -64,7 +64,7 @@ func setupGitea(forge *model.Forge) (forge.Forge, error) {
 	if len(opts.URL) == 0 {
 		return nil, fmt.Errorf("WOODPECKER_GITEA_URL must be set")
 	}
-	log.Trace().Msgf("Forge (gitea) opts: %#v", opts)
+	log.Trace().Msg("Setting up gitea")
```

Comment:
1. But it was a `trace` log, not even `debug`.
2. What if I want to log/debug forge parameters like URL, SkipVerify and OAuthHost?

Reply:
1. Yeah, sure.
2. No :)
2.1. They are go through one transformation [at least](https://github.com/woodpecker-ci/woodpecker/blob/355ebcd508afe107cd5641c4eeb2e4e0c9474795/cmd/server/flags.go#L336). Which [might not work](https://github.com/woodpecker-ci/woodpecker/issues/4503).
2.2. The second transformation might be in some [setup logic](https://github.com/woodpecker-ci/woodpecker/blob/main/server/services/setup.go#L124-L159).
2.3. Also some settings can be changed via API, it appears
![Screenshot 2024-12-10 1](https://github.com/user-attachments/assets/facf384b-df18-4618-af57-ea8b84016770)